Sources/Tracing/Span.swift

```diff
@@ -650,10 +650,26 @@ public struct SpanLink {
 
     /// Create a new `SpanLink`.
     /// - Parameters:
-    ///   - context: The `Baggage` identifying the targeted ``Span``.
+    ///   - baggage: The `Baggage` identifying the targeted ``Span``.
     ///   - attributes: ``SpanAttributes`` that further describe the link. Defaults to no attributes.
     public init(baggage: Baggage, attributes: SpanAttributes = [:]) {
         self.baggage = baggage
         self.attributes = attributes
     }
 }
+
+#if compiler(>=5.6)
+@preconcurrency public protocol _SwiftTracingSendableSpan: Sendable {}
+#else
+public protocol _SwiftTracingSendableSpan {}
+#endif
+
+#if compiler(>=5.6)
+extension SpanAttributes: Sendable {}
+extension SpanAttribute: @unchecked Sendable {}
+extension SpanStatus: Sendable {}
+extension SpanEvent: @unchecked Sendable {} // unchecked because of DispatchWallTime
```

Comment:
Is DispatchWallTime used for locking?

Reply:
No, for the "wall time" time type. Swift's Clocks are missing functionality _and_ we would have to require 5.7 so sadly it's a no go either way... Though I'm still looking into using Swift's clock types

Reply:
but why is it "unchecked because of `DispatchWallTime`"? Is `DispatchWallTime` not marked as Sendable? Shouldn't we use `@preconcurrency` in that case?

Reply:
Right, I just moved to that in [`e3c5bc1` (#83)](https://github.com/apple/swift-distributed-tracing/pull/83/commits/e3c5bc16ad151b54f7c218782f7ab2ac9186362f) :)